examples/cuda/schedulers/detail/bulk.cuh

```diff
@@ -98,7 +98,7 @@ template <class SenderId, std::integral Shape, class FunId>
 
     template <class Self, class Env>
       using completion_signatures =
-        std::execution::__make_completion_signatures<
+        _P2300::execution::__make_completion_signatures<
```

Comment:
Do we want to add a shorthand for `_P2300::execution`?

like `_EXEC` or similar?

Reply:
You're looking at the wrong commit. This PR changes it to `stdexec`. Take a look at the qualification rationale I just added to the PR description and see if you agree.